db/db_impl/db_impl_write.cc

```diff
@@ -1925,10 +1926,10 @@ Status DB::DeleteRange(const WriteOptions& opt,
                        const Slice& begin_key, const Slice& end_key) {
   WriteBatch batch;
   Status s = batch.DeleteRange(column_family, begin_key, end_key);
-  if (!s.ok()) {
-    return s;
+  if (s.ok()) {
+    s = Write(opt, &batch);
   }
-  return Write(opt, &batch);
+  return s;
```

Comment:
I don't see a difference between the changes.

Reply:
Updated.